pod/perlretut.pod

```diff
@@ -60,7 +63,7 @@ many examples.  The first part of the tutorial will progress from the
 simplest word searches to the basic regular expression concepts.  If
 you master the first part, you will have all the tools needed to solve
 about 98% of your needs.  The second part of the tutorial is for those
-comfortable with the basics and hungry for more power tools.  It
+comfortable with the basics, and hungry for more power tools.  It
```

Comment:
This doesn't look right to me.

Going by https://www.grammarly.com/blog/comma-before-and/

It isn't a list of 3 or more items.

They aren't independent - presumably you're looking for more powerful tools since you're comfortable with the basics, though this is a bit of a matter of judgement.

Reply:
Either way seems fine to me